src/dashboard/cd-dashboard-news.vue

```diff
@@ -20,6 +21,9 @@
         <div class="cd-dashboard-news__posts--filler"></div>
       </div>
     </div>
+    <div class="cd-dashboard-news__cta">
+      <a class="cd-dashboard-news__view-all" href="#">View all news</a>
```

Comment:
Translation

Reply:
yep have to put it in for other too, was just waitin till de end